src/cts/src/CtsOptions.h

```diff
@@ -148,9 +149,16 @@ class CtsOptions
   {
     clusteringCapacity_ = capacity;
   }
-  double getBufferDistance() const { return bufDistance_; }
+
+  // BufferDistance is set in microns, but returned in DBU
+  uint32_t getBufferDistance() const { return bufDistance_ * getDbUnits(); }
```

Comment:
why not int?  https://google.github.io/styleguide/cppguide.html#Integer_Types
```In particular, do not use unsigned types to say a number will never be negative.```

In general I would switch this over to dbu internally and put the user micron conversion near UI

Reply:
I did think about that, but these units in particular have default values in microns that makes it hard to initialize without knowing what the DBU conversion is.

I think it would be cleaner to have DBU internally, but it seems difficult to accommodate the default behavior.

Reply:
Perhaps the default could be a sentinel value (eg -1, std::optional) that is converted to dbu at the command start.